### PR TITLE
[UT] add Orc Test with arrow::adapters::orc

### PR DESCRIPTION
**Pull request by zuochunwei**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: cpp/core/memory/ColumnarBatch.h [hunk 1]
```diff
@@ -77,6 +77,10 @@ class ArrowColumnarBatch final : public ColumnarBatch {
     throw gluten::GlutenException("Not implemented GetBytes for ArrowColumnarBatch");
   }
 
+  arrow::RecordBatch* GetRecordBatch() const {
```
Comment (rui-mo) [inline cpp/core/memory/ColumnarBatch.h:80]: Is this function only used for testing? If yes, can we add some notes here?

**Review by rui-mo — APPROVED** [review]

File: cpp/core/memory/ColumnarBatch.h (hunk 1 above)
Comment (zuochunwei) [reply cpp/core/memory/ColumnarBatch.h:80]: ok

File: cpp/velox/tests/OrcTest.cc [hunk 2]
```diff
@@ -0,0 +1,158 @@
+
+#include "benchmarks/BatchStreamIterator.h"
```
Comment (jinchengchenghh) [inline cpp/velox/tests/OrcTest.cc:2]: Sorry for the late review, now we have two options `BUILD_BENCHMARKS` and `BUILD_TESTS`, there can separately build. Now we need to build them at the same time, this is not a good solution. Can you modify the test to remove benchmark link? @zuochunwei

Comment (zuochunwei) [reply cpp/velox/tests/OrcTest.cc:2]: OK, could you help me to process this problem? @yiming-yang

Comment (zuochunwei) [reply cpp/velox/tests/OrcTest.cc:2]: OK, could you help me to process this problem? @yiming-yang

**Pull request merged**